pyproject.toml

```
@@ -178,12 +175,10 @@ check = [
 
 [tool.hatch.envs.style.scripts]
 check = [
-  "black --check --diff --color .",
   "ruff .",
```

still needs to check format
```suggestion
  "ruff format --check --diff .",
  "ruff .",
```

---

thanks, got it